lsstseries/analysis/structurefunction2.py

```diff
@@ -0,0 +1,115 @@
+from scipy.stats import binned_statistic
+import numpy as np
+
+
+def calc_sf2(time, flux, err, bins, band, band_to_calc=None):
+    """Compute structure function squared on one or many bands
+
+    Parameters
+    ----------
+    time : `numpy.ndarray` (N,)
+        Array of times when measurements were taken.
+    flux : `numpy.ndarray` (N,)
+        Array of flux/magnitude measurements.
+    err : `numpy.ndarray` (N,)
+        Array of associated flux/magnitude errors.
+    bins : `numpy.ndarray` (N,)
+        Array of time bins in which to calculate structure function
+    band : `numpy.ndarray` (N,)
+        Array of associated band labels,
+    band_to_calc : `str` or `list` of `str`
+        Bands to calculate StetsonJ on. Single band descriptor, or list
```

Comment:
Adjust description to be structurefunction2-specific

Reply:
Corrected